### ISSUE TITLE
Issue with all day events when using Resources

### ISSUE THREAD

**Issue by DarylBuckle** [intljusticemission/react-big-calendar#727]
**Current Behaviour**

All day events for a Resource span every resource in Week and Day view.

Example code

```
import React from 'react';
import BigCalendar from 'react-big-calendar';
import moment from 'moment';

BigCalendar.setLocalizer(BigCalendar.momentLocalizer(moment))

let resources = [
    {
        resourceId: 1,
        resourceTitle: 'Dave',
    },
    {
        resourceId: 2,
        resourceTitle: 'Steve',
    },
    {
        resourceId: 3,
        resourceTitle: 'Barry',
    },
    {
        resourceId: 4,
        resourceTitle: 'Gary',
    },
]
let events = [
  {
    id: 0,
    title: 'Board meeting',
    start: new Date(2018, 0, 29, 9, 0, 0),
    end: new Date(2018, 0, 29, 13, 0, 0),
    resourceId: 1,
    color:"#000000"
  },
  {
    id: 1,
    title: 'MS training',
    start: new Date(2018, 0, 29, 14, 0, 0),
    end: new Date(2018, 0, 29, 16, 30, 0),
    resourceId: 2,
    color:"#EEEEEE"
  },
  {
    id: 2,
    title: 'Team lead meeting',
    start: new Date(2018, 0, 29, 8, 30, 0),
    end: new Date(2018, 0, 29, 12, 30, 0),
    resourceId: 3,
    color:"#0f61a1",
    allDay:true,
  },
  {
    id: 11,
    title: 'Birthday Party',
    start: new Date(2018, 0, 30, 7, 0, 0),
    end: new Date(2018, 0, 30, 10, 30, 0),
    resourceId: 4,
    color:"#4A10F6",
    allDay:true,
  },
];

class Calendar extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
    }
  }

  render() {
    return (
        <BigCalendar
            events={events}
            defaultDate={new Date(2018, 0, 29)}
            step={60}
            culture={"en-GB"}
            resources={resources}
            resourceIdAccessor="resourceId"
            resourceTitleAccessor="resourceTitle"
        />
    );
  }
}

export default (Calendar);
```

Screenshot below of the output below.

![image](https://user-images.githubusercontent.com/15156674/36196832-dd36ce7c-1169-11e8-9d8f-718a4ff6b026.png)

**Expected behavior**

In this case, the events "Team Lead meeting" and "Birthday Party" should only show for Barry and Gary respectively in the all day row, instead of appearing for every resource on that day. I'm guessing that all day events haven't been handled for resources yet.

P.S Loving this component by the way, fits my requirements almost perfectly. Keep up the good work!

**Comment by brandon-lind** [comment]
Is there an expected fix for this? We're running into this issue as well.

**Comment by sators** [comment]
+1

**Comment by mikeyc7m** [comment]
+1

**Comment by mikeyc7m** [comment]
bump?

**Comment by arecvlohe** [comment]
This looks to be implemented in #423.
